--- sqlite3.c.orig	2019-02-25 19:32:08.000000000 +0300
+++ sqlite3.c	2019-07-31 18:12:21.807566612 +0300
@@ -29486,6 +29486,7 @@ SQLITE_PRIVATE int sqlite3ThreadJoin(SQL
     assert( rc!=WAIT_IO_COMPLETION );
     bRc = CloseHandle((HANDLE)p->tid);
     assert( bRc );
+    (void)bRc; /* Unused. */
   }
   if( rc==WAIT_OBJECT_0 ) *ppOut = p->pResult;
   sqlite3_free(p);
@@ -194877,6 +194878,7 @@ static int sessionPreupdateEqual(
         rc = pSession->hook.xOld(pSession->hook.pCtx, iCol, &pVal);
       }
       assert( rc==SQLITE_OK );
+      (void)rc; /* Unused. */
       if( sqlite3_value_type(pVal)!=eType ) return 0;
 
       /* A SessionChange object never has a NULL value in a PK column */
